doc/dev/developer_guide/tests-unit-tests.rst

```diff
@@ -78,33 +77,41 @@ the `cram task`_.
 
 Tox based testing of python modules
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some of the Python modules in Ceph use `tox <https://tox.readthedocs.io/en/latest/>`_ to run their unit tests.
```

Comment:
I'm a little worried that reStructuredText won't render this section correctly without a blank line between the heading and the first paragraph.

In any case, I think the presence/absence of the newline should be consistent within the document. I do remember religiously inserting a blank line after every heading, so if consistency is desirable it would probably be easier to just insert a blank line here. (As opposed to removing lots of blank lines from elsewhere in the document.)